src/realm/sync/noinst/protocol_codec.hpp

```diff
@@ -341,6 +344,14 @@ class ClientProtocol {
 
                 connection.receive_ident_message(session_ident, client_file_ident); // Throws
             }
+            else if (message_type == "test_command") {
```

Comment:
What is the response if the server does not have the test commands enabled? I'm assuming the body would be some sort of error, if it responds at all? Maybe the error could be translated in Session::receive_test_command_response() instead of passing that on to the test scripts?

Reply:
you'll get an ERROR message back with the error code `ProtocolError::UnknownMessage` and the connection will terminate.

Reply:
And this will come back as a general error message and not via the test_message response?